app/controllers/users_controller.rb

```diff
@@ -30,6 +30,7 @@ def index
     else
       @users = User.joins(:role).where.not(id: User.first.id) #Remove first user
     end
+    @users = User.search(@users, params[:search]) if params[:search]
```

Comment:
It's more important to have clean code than to have the smallest amount of lines. Users shouldn't be malforming the URL parameters, so it's okay to ignore invalid params without a warning.